x-pack/plugins/alerting/server/lib/validate_snooze_date.ts

```diff
@@ -5,9 +5,15 @@
  * 2.0.
  */
 
-export const validateSnoozeDate = (date: string) => {
+export const validateSnoozeEndDate = (date: string) => {
   const parsedValue = Date.parse(date);
   if (isNaN(parsedValue)) return `Invalid date: ${date}`;
   if (parsedValue <= Date.now()) return `Invalid snooze date as it is in the past: ${date}`;
   return;
 };
+
+export const validateSnoozeStartDate = (date: string) => {
+  const parsedValue = Date.parse(date);
+  if (isNaN(parsedValue)) return `Invalid date: ${date}`;
+  return;
```

Comment:
do we care if our snoozeStartDate is in the past? will that be weird or we can accept this behavior.

Reply:
The `snoozeStartDate` will always be in the past because this API takes more than 1ms to execute. I guess we could add a check to make sure the startDate isn't yesterday? But imo it's fine like this